packages/orchestrator-ui-components/src/components/WFOSteps/WFOStepList/WFOStepList.tsx

```diff
@@ -104,6 +131,32 @@ export const WFOStepList = ({ steps = [], startedAt }: WFOStepListProps) => {
             </EuiFlexGroup>
             <>
                 {steps.map((step, index) => {
+                    let prevState = {};
+                    let delta = {};
+
+                    if (index > 0) {
+                        // Todo: decided if we want this here.
+                        //  V1 has a rather big case statement with businness logic here
+                        if (steps[index].status === 'success') {
+                            // Prepare a delta with the last successful step
+                            let prev_index = index - 1;
```

Comment:
camelCase or snake_case? I see both used. I prefer camelCase but at least it should be consistent

Reply:
A mix was used in v1: and this was a copy, not yet ready for reviewing.